src/klib/kvec.h

```diff
@@ -207,6 +207,16 @@ static inline void *_memcpy_free(void *const restrict dest, void *const restrict
   /* 2^x initial array size. */ \
   kvi_resize(v, (v).capacity << 1)
 
+/// fit at least "len" more items
+#define kvi_ensure_space(v, len) \
```

Comment:
the name suggests to me that it would ensure `len` in total, not additional `len` space